media/css/firefox/new/scene1_gb_de.scss

```diff
@@ -0,0 +1,133 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../pebbles/includes/lib";
+
+/* -------------------------------------------------------------------------- */
+// Custom key features section
+
+.features {
+  text-align: center;
+  background: #fff;
+
+  .features-title {
+      color: #000;
+      font-size: 2rem;
+      margin-bottom: 60px;
+  }
+
+  h3 {
+    @include font-size-level4;
+    color: $color-quantum-blue;
+    font-weight: normal;
+    line-height: 1.3;
+    margin: 20px 0 20px 0;
+  }
+
+  ul {
+    margin: 0;
+
+    li {
+      background-position: 50% 20%, 50% 5%;
+      background-repeat: no-repeat;
+      padding-top: 180px;
+
+      &.private {
+        background-image: url("/media/img/firefox/new/icon-mask.svg"), url("/media/img/firefox/new/blob.svg");
+      }
+
+      &.faster {
+        background-image: url("/media/img/firefox/new/icon-clock.svg"), url("/media/img/firefox/new/blob2.svg");
+      }
+
+      &.tracking {
+        background-image: url("/media/img/firefox/new/icon-protection.svg"), url("/media/img/firefox/new/blob3.svg");
+      }
+
+      &.password {
+        background-image: url("/media/img/firefox/new/icon-password.svg"), url("/media/img/firefox/new/blob2.svg");
+      }
+
+      &.sync {
+        background-image: url("/media/img/firefox/new/icon-sync.svg"), url("/media/img/firefox/new/blob.svg");
+      }
+
+      &.switch {
+        background-image: url("/media/img/firefox/new/icon-switch.svg"), url("/media/img/firefox/new/blob3.svg");
+      }
+
+      &:last-child p {
+        margin-bottom: 20px;
+      }
+    }
+  }
+
+  p {
+    @include font-size-level5;
+    margin: 0 auto 40px;
+    max-width: 16em;
+  }
+
+  @media #{$mq-tablet} {
+    padding-top: 40px;
+
+    ul {
+      display: flex;
+      flex-wrap: wrap;
+
+      li {
+        width: 30%;
+        margin: 0 auto;
+      }
+    }
+
+    h3 {
+      margin: 20px 0 20px 0;
+    }
+  }
+}
+
+/* -------------------------------------------------------------------------- */
+// Custom compact callout styles TODO: upgrade to protocol
+
+.call-out-compact {
+    background: url("/media/img/firefox/quantum/fox-tail-header.png") top -750px center no-repeat,
```

Comment:
It might be worth including a fallback for older browsers that don't support CSS gradients / multiple backgrounds. This is how the callout looks in IE9:

<img width="1655" alt="Screenshot 2019-03-15 at 09 13 07" src="https://user-images.githubusercontent.com/400117/54420744-bdb87580-4702-11e9-8686-312348380497.png">

Reply:
I double checked how the protocol callout degrades in IE9 and it should degrade ok-ish. Maybe you could copy & rename the existing callout CSS to see if that works here?

<img width="1646" alt="Screenshot 2019-03-15 at 09 19 24" src="https://user-images.githubusercontent.com/400117/54421034-72eb2d80-4703-11e9-851c-22ed8a8dce98.png">